main.go

```diff
@@ -23,38 +23,52 @@ func main() {
 	log.Println("Successfully established connection to database.")
 
 	log.Println("Starting db schema migration...")
-	driver, _ := postgres.WithInstance(db, &postgres.Config{})
-	m, _ := migrate.NewWithDatabaseInstance(
+	driver, err := postgres.WithInstance(db, &postgres.Config{})
+	if err != nil {
+		log.Panic("Unable to create database instance for migration:", err)
+	}
+
+	m, err := migrate.NewWithDatabaseInstance(
 		"file://migrations/postgres",
 		"postgres", driver)
-	version, _, _ := m.Version()
+	if err != nil {
+		log.Panic("Unable to create Migrate instance for database:", err)
+	}
+
+	version, dirty, err := m.Version()
+	if err != nil {
+		log.Panic("Unable to get current migration version for database:", dirty, err)
+	}
 	log.Println("Current schema version:", version)
+
 	err = m.Up()
 	if err != nil {
-		log.Println("Error while migration:", err)
+		log.Panic("Error while migration:", err)
```

Comment:
@maknahar When the migration is already up-to-date, `m.Up()` will return an error `Error("no change")`. In that case, it cannot be `panic` and server should start.